content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -273,6 +293,51 @@ We may consider adding cross Node preemption in future versions if we find an
 algorithm with reasonable performance. We cannot promise anything at this point, 
 and cross Node preemption will not be considered a blocker for Beta or GA.
 
+## Debugging Pod Priority and Preemption
+
+Pod Priority and Preemption is a major feature that could potentially disrupt
+Pod scheduling if it has bugs.
+
+### Potential problems caused by Priority and Preemption
+
+The followings are some of the potential problems that could be caused by bugs
+in the implementation of the feature. This list is not exhaustive.
+
+#### Pods are preempted unnecessarily
+
+Preemption removes existing Pods from a cluster under resource pressure to make
+room for higher priority pending Pods. If a user gives high priorities to
+certain Pods by mistake, these unintentional high priority Pods may cause
+preemption in the cluster. As mentioned above, Pod priority is specified by
+setting the `priorityClassName` field of `podSpec`. The integer value of
+priority is then resolved and populated to the `priority` field of `podSpec`.
+
+To resolve the problem, `priorityClassName` of the Pods must be changed to use
+lower priority classes or should be left empty. Empty `priorityClassName` is
+resolved to zero by default.
+
+When a Pod is preempted, there will be events recorded for the preempted Pod.
+Preemption should happen only when a cluster does not have enough resources for
+a Pod. In such cases, preemption happens only when the priority of the pending
+Pod (preemptor) is higher than the victim Pods. Preemption must not happen when
+there is no pending Pod, or when the pending Pods have equal or higher priority
+than the victims. If preemption happens in such scenarios, please file an issue.
+
+#### Pods are preempted, but the preemptor is not scheduled
+
+When pods are preempted, they receive their requested graceful termination
+period, which is by default 30 seconds, but it can be any different value as
+specified in the PodSpec. If the victim Pods do not terminate within this period
```

Comment:
nit: I think we should define victim pods first or else it may cause confusion

Reply:
The rest of the document mentions "victim pods" in quite a few places. So, hopefully it is clear what they are.

Reply:
SGTM